addons/godot-firebase/database/reference.gd

```diff
@@ -120,9 +120,9 @@ func get_data() -> Dictionary:
 
 func _get_remaining_path(is_push : bool = true) -> String:
     if !_filter_query or is_push:
-        return _json_list_tag + _query_tag + _auth_tag + Firebase.Auth.auth.idtoken
+        return _json_list_tag + _query_tag + _auth_tag + _auth_obj.auth.idtoken
```

Comment:
Trying out this new code and when using Realtime DB, this `_auth_obj` is null. Same with `_database_obj`. Does it need to be initialized in a different way?

Reply:
Sorry, didn't see this - I think this should be working now, right?

Reply:
> Trying out this new code and when using Realtime DB, this `_auth_obj` is null. Same with `_database_obj`. Which causes an immediate crash. Does it need to be initialized in a different way?

Sorry, didn't see this - I think this should be working now, right?

Reply:
Yep it was fixed in #129